packages/compass-connections/src/components/connections.spec.tsx

```diff
@@ -71,11 +74,13 @@ describe('Connections Component', function () {
       loadConnectionsSpy = sinon.spy(mockStorage, 'loadAll');
 
       render(
-        <Connections
-          onConnected={onConnectedSpy}
-          connectionStorage={mockStorage}
-          appName="Test App Name"
-        />
+        <ConfirmationModalArea>
```

Comment:
For plugins that can be used outside of main rendering point you probably want the main component export of the plugin to be wrapped in this provider, not only in tests, like we discussed related to this change https://github.com/mongodb-js/compass/pull/4079#discussion_r1113059159